Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM icr.io/appcafe/open-liberty:kernel-slim-java11-openj9-ubi
+ARG VERSION=1.0
+ARG REVISION=SNAPSHOT
+
+LABEL \
+  org.opencontainers.image.authors="Your Name" \
+  org.opencontainers.image.vendor="IBM" \
+  org.opencontainers.image.url="local" \
+  org.opencontainers.image.source="https://github.com/OpenLiberty/guide-getting-started" \
+  org.opencontainers.image.version="$VERSION" \
+  org.opencontainers.image.revision="$REVISION" \
+  vendor="Open Liberty" \
+  name="system" \
+  version="$VERSION-$REVISION" \
+  summary="The system microservice from the Getting Started guide" \
+  description="This image contains the system microservice running with the Open Liberty runtime."
```

Comment:
```suggestion
  description="This image contains a sample application that contains a system microservice that retrieves system properties running with the Open Liberty runtime. Uses MicroProfile Config to simulate the status of the microservice, MicroProfile Health to determine the health of the microservice, and MicroProfile Metrics to provide metrics for the microservice."
```

Reply:
I didn't use your description, but I hope the one I've gone with works.